ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -33,6 +34,14 @@ const validationSchema = yup.object().shape({
             })
         ),
     }),
+    cosignCertificates: yup.array().of(
+        yup.object().shape({
+            certificateChainPemEnc: yup.string(),
+            certificatePemEnc: yup.string(),
+            certificateOidcIssuer: yup.string().required('Certificate OIDC issuer is required'),
+            certificateIdentity: yup.string().required('Certificate identity is required'),
```

Comment:
An observation, not necessarily a recommendation.

A common idiom for required string field is `yup.string().trim().required(…) as in `name` on line 28
* Although this has no effect on the value in the form, it means a value of spaces only to fail the `required` validation check.
* For your info, invisible trim caused confusion recently for validation that included `match(/^…$/)` predicate because leading and trailing spaces passed frontend validation, but failed backend validation, because they are in the value. Solution was to remove `trim` and let spaces only. be a match failure instead of a required failure.

While I am over-sharing, notice useless `trim` with `required` for `id` field from finger muscle memory.

Having said all that, you can make an informed decision whether or not to update.

Reply:
I've added the trim portion, should be fine with the current syntax.